tiledb/common/dynamic_memory/test/unit_dynamic_memory.cc

```diff
@@ -74,31 +74,31 @@ TEST_CASE("make_shared - TracingLabel") {
   auto x = make_shared<unsigned short>(label, 5);
 }
 
-TEST_CASE("make_shared_whitebox - TracingLabel") {
-  auto label = TracingLabel(std::string_view("foo", 3));
-  auto x = make_shared_whitebox<unsigned short>(label, 5);
-}
+// TEST_CASE("make_shared_whitebox - TracingLabel") {
+//   auto label = TracingLabel(std::string_view("foo", 3));
+//   auto x = make_shared_whitebox<unsigned short>(label, 5);
+// }
 
 TEST_CASE("make_shared - string constant") {
   auto x = make_shared<unsigned short>("foo", 5);
 }
 
-TEST_CASE("make_shared_whitebox - string constant") {
-  auto x = make_shared_whitebox<unsigned short>("foo", 5);
-}
+// TEST_CASE("make_shared_whitebox - string constant") {
+//   auto x = make_shared_whitebox<size_t>("foo", 5);
```

Comment:
Is there a reason to change the types?

Reply:
No, will revert -- leftover from some attempts to fix the compile error in #4191 (which should go away with sc-33182)